backend/stats/views.py

```diff
@@ -7,49 +7,51 @@
 from rest_framework import status
 from django.db.models import Count, Q
 from User.models import User
+from .user_stats_table import UserTable
+from .leader_board_table import LeaderBoardTable
+from .friend_stats_table import FirendsTable
 
+from .mixins import UserTableMixin
+from rest_framework.response import Response
 
-class StatsView(APIView):
+class StatsView(APIView, UserTableMixin):	
 	def get(self, request):
```

Comment:
I don't know if you and Dorian have talked about whose job it is to show the stats, but it might look better if you return a json response to the frontend that then itself displays the stats data. Graphs and other visualisations of data we def need to do in the backend, of course.

Reply:
No yeah, I'm with you @sakarisalmi
I guess something got lost in communication on Thursday. I do want the "raw" JSON data. Svg is fine for the graphs that show eg wins per day. If absolutely necessary, html elements are fine for play history of a player since they are only used in two places in the same way, but stats of a user is used in different ways all over the place. But even then, I don't see the point of sending html for the history of a player.